src/main/java/net/kodehawa/mantarobot/commands/ActionCmds.java

```diff
@@ -137,18 +66,102 @@ public void register(CommandRegistry cr) {
                 "commands.action.bloodsuck", BLOODSUCK.get(), "commands.action.lonely.bloodsuck", "commands.action.self.bloodsuck", true)
         );
 
-        //lewd();
-        cr.register("lewd", new ImageCmd("Lewd", "T-Too lewd!", "lewd", "lewd", "commands.action.lewd"));
-
         //meow();
         cr.register("meow", new ImageCmd("Meow", "Meows at the specified user.", "meow", MEOW.get(), "commands.action.meow"));
         cr.registerAlias("meow", "mew");
 
-        //nom();
-        cr.register("nom", new ImageCmd("Nom", "*nom nom*", "nom", "nom", "commands.action.nom"));
-
-        //facedesk();
-        cr.register("facedesk", new ImageCmd("Facedesk", "When it's just too much to handle.", "facedesk", "banghead",
-                "commands.action.facedesk", true));
+        if (MantaroData.config().get().hasWeebapiKey()) {
```

Comment:
If you're going here an .isValid() method would also be a good idea. Additionally, one could check wether the API key has the required Permissions on Weeb.sh

Reply:
Refer to https://docs.weeb.sh/